### PR TITLE
MOSIP-12860 Column addtion and dml changes for release scripts

### PR DESCRIPTION
**Pull request by rambhatt1591**

### CODE REVIEW

File: db_release_scripts/mosip_regprc/dml/regprc-transaction_type.csv [hunk 1]
```diff
@@ -0,0 +1,27 @@
+code,descr,lang_code,is_active,cr_by,cr_dtimes
```
Comment (vishwa-vyom) [inline db_release_scripts/mosip_regprc/dml/regprc-transaction_type.csv:1]: For this jira Id, why do we need these changes ?

File: db_release_scripts/mosip_regprc/sql/1.2_regprc-scripts_release.sql [hunk 2]
```diff
@@ -7,10 +7,22 @@
...
+-- Apr-2021		Ram Bhatt	   Added rows to transaction_type.csv
+----------------------------------------------------------------------------------------------------
 \c mosip_regprc sysadmin
 
 ALTER TABLE reg_prc.registration ADD COLUMN resume_timestamp timestamp;
 ALTER TABLE reg_prc.registration ADD COLUMN default_resume_action character varying(50);
 
+---------------------------------------------------------------------------------------------------
+
+ALTER TABLE reg_prc.registration ADD COLUMN resume_remove_tags character varying(256);
```
Comment (vishwa-vyom) [inline db_release_scripts/mosip_regprc/sql/1.2_regprc-scripts_release.sql]: I think it needs to be regprc and not regp_prc

File: db_release_scripts/mosip_regprc/sql/1.2_regprc-scripts_release.sql [hunk 3]
```diff
@@ -7,10 +7,22 @@
...
+
 ----------------------------------------------------------------------------------------------------
+
+TRUNCATE TABLE regprc.transaction_type cascade ;
```
Comment (vishwa-vyom) [inline db_release_scripts/mosip_regprc/sql/1.2_regprc-scripts_release.sql:24]: Why do we need the below statement as per this subtask ?

File: design/data_model/_sources/mosip_regprc.dbm [hunk 4]
```diff
@@ -3,7 +3,7 @@
 CAUTION: Do not modify this file unless you know what you are doing.
          Unexpected results may occur if the code is changed deliberately.
 -->
-<dbmodel pgmodeler-ver="0.9.3" use-changelog="false" last-position="278,42" last-zoom="0.65" max-obj-count="40"
+<dbmodel pgmodeler-ver="0.9.3" use-changelog="false" last-position="116,0" last-zoom="0.65" max-obj-count="40"
```
Comment (vishwa-vyom) [inline design/data_model/_sources/mosip_regprc.dbm:6]: Png file is also not required to be updated ?

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: db_release_scripts/mosip_regprc/dml/regprc-transaction_type.csv (hunk 1 above)
Comment (rambhatt1591) [reply db_release_scripts/mosip_regprc/dml/regprc-transaction_type.csv:1]: Rows to be added for   
WORKFLOW_PAUSE
WORKFLOW_RESUME

Comment (vishwa-vyom) [reply db_release_scripts/mosip_regprc/dml/regprc-transaction_type.csv:1]: Then that case the file should contain only 2 new lines for those 2 types rt ?

**Review by vishwa-vyom — APPROVED** [review]

**Pull request merged**